### ISSUE TITLE
Import notes from Bear

### ISSUE THREAD

**Issue by kimaldis** [HappenApps/Quiver#1106]
Any way to import a note archive from Bear? I can't see anything immediately obvious in either's menus

**Comment by thehilll** [comment]
If you export the Bear notes as markdown and then import markdown within Quiver it works pretty well.

**Comment by kimaldis** [comment]
yeah, what I ended up doing. I was hoping more for something that would keep the hierarchical structure. Although I appreciate that's a stretch.

**Comment by moyaghini** [comment]
> If you export the Bear notes as markdown and then import markdown within Quiver it works pretty well.

But how to copy over the inline images? I found no easy way of doing that.
